clusterloader2/pkg/execservice/exec_service.go

```diff
@@ -63,12 +70,16 @@ func InitFlags(c *config.ExecServiceConfig) {
 		"Whether to enable exec service that allows executing arbitrary commands from a pod running in the cluster.",
 	)
 	flags.StringEnvVar(
-		&c.DeploymentYaml,
+		&unusedDeploymentYamlFlagValue,
 		"exec-deployment-yaml",
 		"EXEC_DEPLOYMENT_YAML",
 		"pkg/execservice/manifest/exec_deployment.yaml",
 		"Path to execservice deployment yaml.",
 	)
+	err := flags.MarkDeprecated("exec-deployment-yaml", "exec_deployment.yaml is embed during a build now.")
```

Comment:
I'm not sure anyone uses this flag besides myself (I added it so I could build cl2 using nix). I don't think it'd be a huge problem if we removed it.

Reply:
You mean remove completely?

I can do that! :)